src/DependencyManagement/DependencyManager.cs

```diff
@@ -99,6 +99,11 @@ internal string Initialize(ILogger logger)
                 // Parse and process the function app dependencies defined in the manifest.
                 _dependenciesFromManifest = _storage.GetDependencies().ToArray();
 
+                if (_dependenciesFromManifest.Length >= 1 && WorkerEnvironment.IsLinuxConsumptionOnLegion())
```

Comment:
If you put block under `if (!_dependenciesFromManifest.Any()) { ... }`, you will also be able to remove the first part of the condition.

Reply:
Good point. Fixed.